spec/system/employee_earned_points_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+require 'factory_bot_rails'
+
+RSpec.describe 'Working on earned points', type: :system do
```

Comment:
Specs looks good, but you are not really working with points, but kudos, so I would rather add those examples to te kudo crud spec - when creating kudo it decreases number of.., and increases number of...